### DISCUSSION TITLE
How can I install a specific version of debian?

### DISCUSSION THREAD

**Discussion by maxnadig** [microsoft/WSL#7806] [Q&A]
With WSL I only seem to be able to install whatever Debian version that is currently supported. In my case this is version it was 11. I would like to use version 10 though. Did anyone figure out how to install it with WSL?

Thanks for the help!

**Comment by Biswa96** [comment]
The GNU/Linux distributions for WSL are provided their own developers. You should ask Debian developers about the official way of doing this. I generally use Docker images for testing. Link https://github.com/debuerreotype/docker-debian-artifacts/tree/dist-amd64
